PalindromString.java

```diff
@@ -0,0 +1,29 @@
+import java.util.Scanner;
+
+class PalindromString
+{
+	public static void main(String args[])
+	{
+		Scanner in =  new Scanner(System.in);
+		System.out.println("Enter any string: ");
+		String str = in.nextLine();
+		if(isPalindrom(str))
+			System.out.println(str + " is a palindrom string");
+		else
+			System.out.println(str + " is not a palindrom string");
+	
+	}
+
+	static boolean isPalindrom(String str)
+	{
+		int i=0,j=str.length()-1;		
```

Comment:
use for loop to minimize lines 19,24,25.

Reply:
done using for loop